tests/integration/customers.test.js

```diff
@@ -854,6 +854,28 @@ describe('CUSTOMERS ROUTES', () => {
 
         expect(res.statusCode).toBe(403);
       });
+
+      it('should not archived customer with non-billed, cancelled but to invoice interventions', async () => {
+        const res = await app.inject({
+          method: 'PUT',
+          url: `/customers/${customersList[13]._id}`,
+          payload: { archivedAt: '2021-01-16T14:30:19' },
+          headers: { Cookie: `alenvi_token=${authToken}` },
+        });
+
+        expect(res.statusCode).toBe(403);
+      });
+
+      it('should archived customer with non-billed, cancelled but not to invoice interventions', async () => {
```

Comment:
que penses tu de mettre ce it juste apres `it('should archive a customer if all interventions are billed', async () => {` ?